frontend/src/locales/de.json

```diff
@@ -189,6 +203,9 @@
   },
   "views": {
     "activity": {
+      "sidebarProgram": {
+        "name": "Tagesübersicht"
+      },
       "sideBarProgram": {
         "title": "Tagesübersicht"
       }
```

Comment:
Wie können wir sicherstellen das in Zukunft das nicht mehr passiert?

Reply:
Das liegt daran dass in zwei unterschiedlichen JSON Files bereits zwei unterschiedliche Schreibweisen vorhanden waren. In Zukunft wird man neue Features programmieren und nur in einem JSON die Übersetzungen ergänzen, dann übernimmt Lokalise denselben Key einfach in die anderen JSONs sobald er übersetzt wird.